src/lucky/params.cr

```diff
@@ -21,6 +21,13 @@ class Lucky::Params
   def initialize(@request : HTTP::Request, @route_params : Hash(String, String) = empty_params)
   end
 
+  def self.from_hash(data : Hash(String, String | Array(String))) : self
+    request = HTTP::Request.new("GET", "/", body: "", headers: HTTP::Headers.new)
+    request.query = URI::Params.encode(data)
```

Comment:
This is sort of a hack because I couldn't think of a better way. This should be fine for uri encoded params, but would fail for json or multipart... My guess is we could maybe make some overloads? I'm open to ideas on a better way